websites/G/Geotastic/dist/metadata.json

```
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Nate",
+		"id": "196817730257551360"
+	},
+	"service": "Geotastic",
+	"description": {
+		"en": "Geotastic is a geographical location guessing game created by Eduard But from Edutastic Games.",
+		"fr": "Geotastic est un jeu de devinette de emplacement géographique créé par Eduard But de Edutastic Games."
```

This translation would be the most appropriate in my opinion 
```suggestion
		"fr": "Geotastic est un jeu de devinette de localisation géographique créé par Eduard But de Edutastic Games."
```